test/src/unittests/tonal/test_pitchcrepe.py

```diff
@@ -47,23 +47,41 @@ def testRegression(self):
     def testEmptyModelName(self):
         # With empty or undefined model names the algorithm should skip the configuration
         # without errors.
-        CREPE()
-        CREPE(graphFilename='')
-        CREPE(graphFilename='', input='')
-        CREPE(graphFilename='', input='wrong_input')
+        PitchCREPE()
+        PitchCREPE(graphFilename='')
+        PitchCREPE(graphFilename='', input='')
+        PitchCREPE(graphFilename='', input='wrong_input')
 
     def testInvalidParam(self):
         model = join(testdata.models_dir, 'crepe', 'crepe-tiny-1.pb')
-        self.assertConfigureFails(CREPE(), {'graphFilename': model,
-                                            'input': 'wrong_input_name',
-                                            'output': 'model/classifier/Sigmoid',
-                                           })  # input do not exist in the model
-        self.assertConfigureFails(CREPE(), {'graphFilename': 'wrong_model_name',
-                                            'input': 'frames',
-                                            'output': 'model/classifier/Sigmoid',
-                                           })  # the model does not exist
-
-suite = allTests(TestCREPE)
+        self.assertConfigureFails(PitchCREPE(), {'graphFilename': model,
+                                                 'input': 'wrong_input_name',
+                                                 'output': 'model/classifier/Sigmoid',
+                                                })  # input do not exist in the model
+        self.assertConfigureFails(PitchCREPE(), {'graphFilename': 'wrong_model_name',
+                                                 'input': 'frames',
+                                                 'output': 'model/classifier/Sigmoid',
+                                                })  # the model does not exist
+
+    def testEmptySignal(self):
+        model = join(testdata.models_dir, 'crepe', 'crepe-tiny-1.pb')
+        self.assertComputeFails(PitchCREPE(graphFilename=model), [])
```

Comment:
Is it conceptually fine for us that it fails on empty input? Is this consistent with other pitch estimation algorithms such as PitchYinProbabilistic? What is the minimum length required not to fail and is this information easily available to a user?

Reply:
Failing in empty input is more consistent with other pitch algorithms:
```
test/src/unittests/sfx/test_pitchsalience.py:
  133  
  134:     def testEmpty(self):
  135          self.assertComputeFails(PitchSalience(), **[])**

test/src/unittests/tonal/test_pitchyin.py:
  26  
  27:     def testEmpty(self):
  28          self.assertComputeFails(PitchYin(), [])

test/src/unittests/tonal/test_pitchyinfft.py:
  26  
  27:     def testEmpty(self):
  28          self.assertComputeFails(PitchYinFFT(), [])

test/src/unittests/tonal/test_pitchyinprobabilistic.py:
  25  
  26:     def testEmpty(self):
  27          self.assertComputeFails(PitchYinProbabilistic(), [])
```